ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -405,6 +206,14 @@ open class BadgeView: UIView, TokenizedControlInternal {
         }
 
         updateFonts()
+
+        // Update appearance whenever `tokenSet` changes.
+        tokenSetSink = tokenSet.objectWillChange.sink { [weak self] _ in
+            // Values will be updated on the next run loop iteration.
+            DispatchQueue.main.async {
+                self?.updateColors()
```

Comment:
we don't need to update fonts when tokenset change?

Reply:
We do, good catch.